assets/javascripts/discourse/templates/components/topic-chat-float.hbs

```diff
@@ -38,31 +38,28 @@
         {{/if}}
 
         {{#if chat.activeChannel}}
-          {{#link-to
-            infoTabRoute
-            chat.activeChannel.id
-            chat.activeChannel.title
-            class="topic-chat-drawer-header__title"
-          }}
-            <div class="topic-chat-drawer-header__top-line">
-              {{#chat-channel-title channel=chat.activeChannel}}
-                {{#if (and (not expanded) unreadCount)}}
-                  <span class="chat-unread-count">{{unreadCount}}</span>
-                {{/if}}
-              {{/chat-channel-title}}
-            </div>
-
-            {{#if displayMembers}}
-              <div class="topic-chat-drawer-header__bottom-line">
-                <span class="chat-channel-member-count">
-                  {{i18n
-                    "chat.channel.memberships_count"
-                    count=chat.activeChannel.memberships_count
-                  }}
-                </span>
+          {{#if expanded}}
+            {{#link-to
+              infoTabRoute
+              chat.activeChannel.id
+              chat.activeChannel.title
+              class="topic-chat-drawer-header__title"
+            }}
+              <div class="topic-chat-drawer-header__top-line">
+                {{chat-channel-title channel=chat.activeChannel}}
               </div>
-            {{/if}}
-          {{/link-to}}
+            {{/link-to}}
+          {{else}}
+            <div role="button" {{action "toggleExpand"}} class="topic-chat-drawer-header__title">
```

Comment:
I would love to see an acceptance test here because the behaviour of clicking on `topic-chat-drawer-header__title` changes depending on whether the minimized chat view is expanded or collapsed.

Reply:
probably out of scope of things I can figure out how to do today :) but I am happy just to hand this over to Joffrey or someone else to finish, if you feel having a test here is very important.

Reply:
I will add this test.

Reply:
Done!